integration_test/workflow_tests/run_tests.py

```diff
@@ -0,0 +1,34 @@
+import unittest
```

Comment:
Could add a shebang line `#!/usr/bin/env python` and set the file to executable as an option.

Reply:
?